src/components/ftbot/TradeListNav.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <b-list-group>
+    <b-list-group-item
+      v-for="trade in trades.sort((a,b) => a.open_timestamp < b.open_timestamp)"
+      :key="trade.open_timestamp"
+      button
+      class="d-flex justify-content-between align-items-center py-1"
+      :title="`${trade.pair}`"
+      :active="trade.open_timestamp === selectedTrade.open_timestamp"
+      @click="selectedTrade = trade; $emit('trade-select',trade);"
```

Comment:
selectedTrade is defined as integer - so assigning a Trade object is a typescript violation ...

Reply:
Right, a leftover from when I just used the index, will fix too.